requirements.txt

```diff
@@ -6,3 +6,30 @@ django-celery-results==1.0.1
 djangorestframework==3.8.2
 phonenumbers==8.9.1
 django-controlcenter==0.2.6
+amqp==2.3.2
```

Comment:
Let's clean up some of these dependencies in favor of the root packages that require them - that'll make sure during upgrades that we don't get stuck on previous dependency versions.